draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1277,6 +1335,30 @@ To indicate that the content is an SD-JWT:
 * Change Controller: IETF
 * Provisional registration?  No
 
+<br/>
+To indicate that the content is an SD-JWT-KB:
```

Comment:
Probably makes sense to add this but I am really not looking forward to folks wanting to have `sd-jwt-kb` as the value of `typ` header in the SD-JWT itself...

Reply:
when is this media type intended to be used? when sending sd-jwt-kb in an http request? not sure I support defining this...

Reply:
maybe we can add this in a separate PR..?

Reply:
We could add a note here saying that this media type is not for use in the issuer-signed part itself.

Reply:
Done!